### DISCUSSION TITLE
CQube bootcamp

### DISCUSSION THREAD

**Discussion by DevashishRaghav** [Sunbird-cQube/community#790] [Announcements]
### Case collaterals

Dear attendees
Here are all the collaterals for the guided & hands-on case based exercises. Please download the case specific zip as per the discussion in the bootcamp.
[Case.1.zip](https://github.com/Sunbird-cQube/community/files/11855953/Case.1.zip)
[Case 2.zip](https://github.com/Sunbird-cQube/community/files/11852782/Case.2.zip)
[Case 3.zip](https://github.com/Sunbird-cQube/community/files/11852793/Case.3.zip)

**Comment by DevashishRaghav** [comment]
### Instructions to connect to the server

**Ubuntu:** 
open terminal and use the following command
ssh -i <pem_file_path> -o ServerAliveInterval=60 ubuntu@<public_ip>

**Windows:**
Install gitbash using[ https://git-scm.com/downloads](https://git-scm.com/downloads)
ssh -i <pem_file_path> -o ServerAliveInterval=60 ubuntu@<public_ip>

**Port forwarding:**
ssh -L <port>:localhost:<port> -i <pem_file_path>  -o ServerAliveInterval=60 ubuntu@<public_ip>

**Connecting to minio:**
ssh -L 9000:localhost:9000 -i <pem_file_path>  -o ServerAliveInterval=60 ubuntu@<public_ip>
localhost:9000
username: minioadmin
password: minioadmin

-----------------------------------------------------------------------------------------------------------------

### VM details table-wise

**Table1**:
Public ip: 3.110.65.210
api_endpoint: cqube-table1.tibilprojects.com

**Table2:**
Public ip: 65.1.129.140
api_endpoint: cqube-table2.tibilprojects.com

**Table3:**
Public ip: 13.200.13.148
api_endpoint: cqube-table3.tibilprojects.com

**Table4:**
Public ip: 15.207.120.158
api_endpoint: cqube-table4.tibilprojects.com

**Table5:**
Public ip: 3.110.68.234
api_endpoint: cqube-table5.tibilprojects.com

**Table6:**
Public ip: 15.207.121.244
api_endpoint: cqube-table6.tibilprojects.com

table7
ip: 3.110.61.215
url: [cqube-table7.tibilprojects.com](http://cqube-table7.tibilprojects.com/)

keycloak creds
username: admin
paswd: Admin@123

Dashboard creds
username: admin
paswd: Admin@123




table8
ip: 15.206.202.206
url: [cqube-table8.tibilprojects.com](http://cqube-table8.tibilprojects.com/)

keycloak creds
username: admin
paswd: Admin@123

Dashboard creds
username: admin
paswd: Admin@123


table9:
ip: 3.110.201.164
url: cqube-table9.tibilprojects.com

**Comment by ChakshuGautam** [comment]
Raw File - https://raw.githubusercontent.com/ChakshuGautam/cQube-ingestion/master/impl/c-qube/fixtures/2023-01-11.csv

**Comment by DevashishRaghav** [comment]
Link to postman collections:

https://drive.google.com/drive/folders/12Xvt6a6Qh1lVoFstD7HCujE7PcJ7vMqv

**Comment by RNarmatha** [comment]
To download the NIFI in Windows . [Link](https://archive.apache.org/dist/nifi/1.12.1/nifi-1.12.1-bin.zip)
